### PR TITLE
doc: add link to side project and create a debugging page

### PR DESCRIPTION
**Pull request by freeboub**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,5 +13,7 @@ Version 6.0.0 is introducing dozens of breaking changes, mostly through updated
 - [Documentation](API.md)
 - [Changelog](CHANGELOG.md)
 - [Contribution guide](CONTRIBUTING.md)
+- [Usefull Side Project](./docs/PROJECTS.md)
```
Comment (LunatiqueCoder) [inline README.md:16]: Perhaps such a link could be useful in [API.md at L428](https://github.com/react-native-video/react-native-video/blob/master/API.md?plain=1#L428)?  

EDIT: L428 should also be under `controls` prop I believe? And not under `contentStartTime` prop.

File: docs/PROJECTS.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+This page links other open source projects which can be usefull for your player implementation
+
+# UI over react-native-video
+ - [react-native-video-controls](https://github.com/itsnubix/react-native-video-controls): First reference player UI
+ - [react-native-media-console](https://github.com/criszz77/react-native-media-console): React-native-video-controls updated and rewritten in typescript
+ - [react-native-corner-video](https://github.com/Lg0gs/react-native-corner-video): A floating video player
+
+# Other tools
+ - [react-native-corner-video](https://github.com/tanguyantoine/react-native-music-control): A toolbox to control player over media session
```
Comment (Lg0gs) [inline docs/PROJECTS.md]: Line 9, the name should be _react-native-music-control_

File: README.md (hunk 1 above)
Comment (freeboub) [reply README.md:16]: OK, done, thank you for the catch !

File: docs/PROJECTS.md (hunk 2 above)
Comment (freeboub) [reply docs/PROJECTS.md]: OK, done, thank you

**Review by Lg0gs — APPROVED** [review]

**Review by LunatiqueCoder — APPROVED** [review]

**Review by Lg0gs — APPROVED** [review]

**Pull request merged**